DevOps/Readme.md

```diff
@@ -138,10 +138,14 @@
 ## Cloud Providers
 
 - [Civo](https://www.civo.com/docs)
+- [Getting started with Civo - Youtube](https://www.youtube.com/watch?v=mHtCkmshfks)
```

Comment:
Since you are specifically adding Video resources, you can also consider to create a sub section under Cloud providers for video resources. It would be easier for user to navigate that way

Reply:
Sure